src/telegrafs/actions/thunks.ts

```diff
@@ -168,8 +204,28 @@ export const removeTelegrafLabelsAsync = (
   labels: Label[]
 ): AppThunk<Promise<void>> => async (dispatch): Promise<void> => {
   try {
-    await client.telegrafConfigs.removeLabels(telegrafID, labels as ILabel[])
-    const telegraf = await client.telegrafConfigs.get(telegrafID)
+    if (labels.length === 0) {
+      throw new Error('No label found to delete')
+    }
```

Comment:
Again, I would check to see if the UI only ever passes in a label or if we anticipate an array and just refactor this from there